modules/sdk/gradle-plugins-workspace/src/gradleTest/providedModules/build.gradle

```diff
@@ -9,7 +9,7 @@ runGradleTest {
 	dependsOn initBundle
 
 	doLast {
-		configurations.providedModules.dependencies.each {
+		configurations.compileOnlyModules.dependencies.each {
```

Comment:
why did you change this?

Reply:
I didn't change this explicitly, I suspect it was done by source formatter.

Reply:
got it... so change it back and add exclusion rule if we need. thanks.